dds/DCPS/Serializer.inl

```diff
@@ -1522,6 +1533,47 @@ void serialized_size_ulong(const Encoding& encoding, size_t& size, size_t count)
   max_serialized_size_ulong(encoding, size, count);
 }
 
+ACE_INLINE
+void serialized_size_delimiter(const Encoding& encoding, size_t& size)
+{
+  if (encoding.xcdr_version() == Encoding::XCDR_VERSION_2) {
+    max_serialized_size_ulong(encoding, size);
+  }
+}
+
+ACE_INLINE
+void serialized_size_parameter_id(
+  const Encoding& encoding, size_t& size, size_t& xcdr1_running_size)
+{
+  const Encoding::XcdrVersion xcdr = encoding.xcdr_version();
+  if (xcdr == Encoding::XCDR_VERSION_1) {
+    encoding.align(size, 4);
+    size += uint16_cdr_size * 2;
+    // TODO(iguessthislldo): Extended PID
+
+    // Save and Zero Size to Reset the Alignment
+    xcdr1_running_size += size;
+    size = 0;
+  } else if (xcdr == Encoding::XCDR_VERSION_2) {
+    encoding.align(size, uint32_cdr_size);
+    size += uint32_cdr_size;
+    // TODO(iguessthislldo) LC
+  }
+}
+
+ACE_INLINE
+void serialized_size_sentinel_parameter_id(
+  const Encoding& encoding, size_t& size, size_t& xcdr1_running_size)
+{
+  if (encoding.xcdr_version() == Encoding::XCDR_VERSION_1) {
+    encoding.align(size, uint16_cdr_size); // TODO(iguessthislldo): Not align 4?
```

Comment:
Thanks for clearing that up for me.

Reply:
We should make sure to clear it up in the spec too.  "May be added" is a little confusing.  And rule `(23)` is just wrong.

Reply:
Issue number DDSXTY14-23 (just a coincidence that it's 23) created for this.  Please add a comment with that issue identifier just in case the issue gets resolved in a different way.